netbox/netbox/settings.py

```diff
@@ -179,6 +179,7 @@
 STORAGE_CONFIG = getattr(configuration, 'STORAGE_CONFIG', {})
 TIME_ZONE = getattr(configuration, 'TIME_ZONE', 'UTC')
 TRANSLATION_ENABLED = getattr(configuration, 'TRANSLATION_ENABLED', True)
+BRANCHING_BACKEND = getattr(configuration, 'BRANCHING_BACKEND', None)
```

Comment:
IMO it does not make sense to expose this as a configuration variable, as this functionality is unique to the netbox-branching plugin.